### PR TITLE
feat(docs): action example in main readme

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: action/README.md [hunk 1]
```diff
@@ -19,11 +19,18 @@ Actions" deployment mode. You can do this in your project settings on
...
 
 ```yml
+name: Deploy
+
+on: push
+
 jobs:
   deploy:
+    runs-on: ubuntu-latest
+
```
Comment (satyarohith) [inline action/README.md:29]:
```suggestion
```

**Review by ry — COMMENTED** [review]
Thanks!

I think we should also have this example in the root README.md ... I came to deployctl the other day trying to find this and it took me a while to find action/README.md

**Comment by roj1512** [comment]
@ry Give me another review.

**Review by satyarohith — APPROVED** [review]
LGTM.

Thanks for your contribution!

**Pull request merged**